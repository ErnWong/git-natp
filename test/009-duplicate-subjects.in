A---C---A master
 \ / \ /
  B   A
