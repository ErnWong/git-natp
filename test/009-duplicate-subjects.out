A
B A
C A B
A_ C
A__ C A_
[master] A__
